README.md

```diff
@@ -16,48 +16,51 @@ theme/
   ...remaining theme files
 ```
 
-## Project Setup
-
+## Prereqs
 1. Make sure you have [Shopify's themekit](https://shopify.github.io/themekit/) installed globally, `npm i -g themekit`.
 2. Make sure you have [Prepros](https://prepros.io/) version 7 or later installed. 
-3. Put your theme files into the `theme` directory
-4. Get Shopify API Access via a priate app. Instructions can be found under the "Get API Access" setion on [this page](https://shopify.github.io/themekit/). Then copy the password to your private app.
-5. Put your Shopify _store_ and private app _password_ in the `theme/config.yml` file.
-6. Locate your _theme_id_:
-   1. Go to the `Themes` setion of your Shopify store.
-   2. Click on the `Actions` menu next to the theme you are developing and click on `Edit code`.
-   3. You'll now see your theme_id (series of numbers) at the end of the URL. Copy it and paste it your theme/config.yml` file).
-7. If you're not using the current published theme you'll need this extra step to get a proxy URL for Prepros that includes your theme id:
+
+## Project Setup
+
+1. In your shopify store, create two duplicates of your production theme. One will be used while actively developing the theme, while the other will be used to snapshot features/fixes that are ready for the client to preview. Identify the duplicates with a name prefix like  `Seamonster Dev ::`  and `SeaMonster Stage ::`.
```

Comment:
lets specifically use these prefixes:

- SMS [STAGE]
- SMS [MAIN]

Reply:
Or do you think SMS [PROD] would be best?

Reply:
Well the client will typically already have a theme in place. Lets say that theme is Debut. The my thinking was:

- Github main branch -> Debut
- Github staging branch -> SMS [STAGE] :: Debut
- Github feature-x branch -> SMS [DEV] :: Debut

Sounds like your idea is a bit different, but I don't see how the mapping would line up.

Reply:
@nireno Yeah, I think that's fine. Can we use the prefixes you defined here in the comments then? Over the ones used in the readme now?

Reply:
@logandeancall Done.

Reply:
![shopify theme names](https://user-images.githubusercontent.com/5152553/133326568-fac0fed6-49e1-41e9-8f78-4fac002456ba.png)
I'm not seeing the update here

Reply:
@logandeancall It should be there. Can you take another look?

Reply:
Yeah, weird, I can see the updates now. I made one more tweak, I changed it to use brackets instead of double colons for dev/stage theme names b/c it matches what we've been doing in our latest Shopify stores.